media/sourcebuffer-changetype.js

```diff
@@ -0,0 +1,50 @@
+const mediaSource = new MediaSource();
+const audio = document.querySelector('audio');
+audio.src = URL.createObjectURL(mediaSource);
+
+mediaSource.addEventListener('sourceopen', function() {
+  URL.revokeObjectURL(audio.src);
+  feedAudioElement(0 /* index */);
```

Comment:
Perhaps for readability, s/index/resources_index/ and move the "const resources = ...;" block to before this event listener?